méli-mélo/2024-10-datatable-utilities/datatable-utilities-doc-en.html

```diff
@@ -0,0 +1,97 @@
+---
+title: DataTable Utilities - Documentation
+language: en
+description: Utility to Manipulate JSON Data.
+tag: DataTable
+altLangPage: datatable-utilities-doc-fr.html
+dateModified: 2024-11-01
+---
+
+<div class="wb-prettify all-pre"></div>
+
+<div>
+    <a href="index-en.html">Back</a>
+
+    <h2>Configuration options</h2>
+
+    <p>Document the public configuration options that can be used by implementers or developers.</p>
+    <p>Enable the plugin for the Target Table by adding wb-tables-utility to the &lt;table&gt; element <code>&lt;table class=&quot;table wb-tables wb-tables-utility&quot;&gt; data-wb-tables-utility=&quot;{...}&quot;</code></p>
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Option</th>
+                <th>Description</th>
+                <th>How to configure</th>
+                <th>Values</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>debug</td>
+                <td>Print Debug messages to the console</td>
+                <td><code>data-wb-tables-utility="{&amp;quot;debug&amp;quot;:true}"</code></td>
+                <td>
+                    <dl>
+                        <dt>false (default)</dt>
+                        <dd>Suppress Printing of debug messages.</dd>
+                        <dt>true</dt>
+                        <dd>Show debug messages to the console.</dd>
+                    </dl>
+                </td>
+            </tr>
+            <tr>
+                <td>filteredsum</td>
+                <td>Column Totals to reflect Filtered DataTable Data</td>
+                <td><code>data-wb-tables-utility="{&amp;quot;filteredsum&amp;quot;:false}"</code></td>
+                <td>
+                    <dl>
+                        <dt>true (default):</dt>
+                        <dd>As Data is Filtered adjust Totals to match filtered Data.</dd>
+                        <dt>false:</dt>
+                        <dd>Show Grand Total Regardless of data filtered.</dd>
+                    </dl>
+                </td>
+            </tr>
+            <tr>
+                <td>wb-col-sum</td>
+                <td>When applied to a column, a column footer total is calcualted</td>
```

Comment:
Just to keep it consistent.
```suggestion
                <td>When this class is applied to a column, a column footer total is calculated</td>
```